### ISSUE TITLE
Git for WIndows 2.35.2 broken on my system

### ISSUE THREAD

**Issue by Robbbert** [git-for-windows/git#3788]
- [ ] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?
 
2.35.2 (32-bit)

```
$ git --version --build-options

Too late, I already reverted.
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

Windows 7, 64-bit.

```
$ cmd.exe /c ver

Microsoft Windows [Version 6.1.7601]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

Defaults

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Too late, I already reverted
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

The Windows 7 machine acts as a gateway to the development machine, windows 10, which is on the internal network. Mapped drives are used.

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Tortoisegit

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

I tried to commit to the repository on github, but it looks like other commands trigger the problem.

 - What did you expect to occur after running these commands?

I expected tortoisegit to list the new and changed files, then i could click on commit

 - What actually happened instead?

```
 - warning: encountered old-style '/bin/msmtp.exe' that should be '%(prefix)/bin/msmtp.exe'
```

Then it said I should do a git command to mark the directory as safe, but that also failed with the above error.

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

I had to push something in a hurry, so I reverted to 2.35.1(2), which works. I'm not going to "upgrade" until you're sure the problem is fixed. Also I don't want to have to buy a new computer because you don't want to support windows 7 any more.

**Comment by rimrul** [comment]
> The Windows 7 machine acts as a gateway to the development machine, windows 10, which is on the internal network. Mapped drives are used.

This sounds like this could be a duplicate of #3786.

**Comment by dscho** [comment]
> This sounds like this could be a duplicate of #3786.

Yep, I think so, too, and it should be fixed by v2.35.3.
